components/ratgdo/ratgdo.cpp

```diff
@@ -428,10 +404,16 @@ namespace ratgdo {
                     if (*this->openings != 0) { // have openings
                         return RetryResult::DONE;
                     } else {
+                        if (r==0) { // failed to sync probably rolling counter is wrong, notify
```

Comment:
I initially thought that it's enough to trigger notification on status failed branch, but it occurred to me that a status message could be sent by the garage opener for other reasons than because we asked for it, so trigger the notification on openings sync failure as well.

Reply:
Do we need to reset to false ever?

Also should it wait a few seconds to send the service call so HA has time to connect to the device after it reboots?

Reply:
> Do we need to reset to false ever?

Only if we want to trigger notification again, which we shouldn't need to. 

> Also should it wait a few seconds to send the service call so HA has time to connect to the device after it reboots?

We're already "waiting" 17s, which is how long it takes to retry 10 times with a start of 300ms and a back-off factor of 1.5. We could increase the initial wait to 400ms or 500ms (which is probably a good idea anyway) and then we would end up "waiting" 23s or 28s before triggering the notification.

Reply:
> We could increase the initial wait to 400ms or 500ms (which is probably a good idea anyway) and then we would end up "waiting" 23s or 28s before triggering the notification.

That sounds good

Reply:
> Only if we want to trigger notification again, which we shouldn't need to.
> 
> >

👍